docs/src/modules/ROOT/partials/concepts-intro.adoc

```diff
@@ -44,6 +44,60 @@ You create a continuous incoming stream of events with the xref:java:http-endpoi
 You create a stream processor to analyze and act against a stream of data with the xref:java:consuming-producing.adoc[Consumer] component.
 |===
 
+== Components
+
+You use xref:reference:glossary.adoc#component[Akka _Components_] to build your application. These Components are crucial for ensuring responsiveness. Here is a brief overview of each. Except endpoints, Akka components will live in your `application` package.
```

Comment:
saw the "Here is a brief" and "Akka components" sentences so the paragraph ends with "here is a brief"